state-chain/chains/src/dot.rs

```diff
@@ -1119,6 +1123,16 @@ mod test_polkadot_extrinsics {
 			"encoded extrinsic: {:?}",
 			extrinsic_builder.get_signed_unchecked_extrinsic().unwrap().encode()
 		);
+
+		let new_replay_protection = PolkadotReplayProtection {
+			nonce: 13,
+			signer: account_id_1,
+			genesis_hash: Default::default(),
+		};
+
+		extrinsic_builder.refresh_replay_protection(new_replay_protection.clone());
+
+		assert_eq!(new_replay_protection, extrinsic_builder.replay_protection);
```

Comment:
Careful - this test has the #[ignore] annotation so is never actually run.

I think it was used just to output some test payloads (that's why there are all the print statements).

Better to create a dedicated test for refreshing the replay protection. 

Also we should test that the signature is removed:

```suggestion
		assert_eq!(new_replay_protection, extrinsic_builder.replay_protection);
		assert!(!extrinsic_builder.is_signed());
```